docs/api/dialog.md

```diff
@@ -185,7 +185,7 @@ The `filters` specifies an array of file types that can be displayed, see
 
 * `browserWindow` [BrowserWindow](browser-window.md) (optional)
 * `options` Object
-  * `title` String (optional)
+  * `title` String (optional) - Title of the dialog, some platforms will not show it.
```

Comment:
This should specify the platforms on which it works. 

See
https://github.com/electron/electron/blob/2865d161b720e209f29ca84858492e095dc3e161/docs/api/dialog.md#L96-L103 for prior art.